### ISSUE TITLE
Create org for this extension

### ISSUE THREAD

**Issue by inson1** [OhMyGuus/I-Still-Dont-Care-About-Cookies#10714]
So there could be all projects on place. It fives off more "professional" vibes. And for future so there could be more members etc.

**Comment by OhMyGuus** [comment]
Yeah was already looking into this before, mostly because it makes permissions easier to handle in github as well. Since now it's either all permissions or none, which is a bit annoying.
